### ISSUE TITLE
Latest update , zfs manager fails to launch 

### ISSUE THREAD

**Issue by Ondjultomte** [optimans/cockpit-zfs-manager#24]
![image](https://user-images.githubusercontent.com/16661547/104128950-a7b18580-536a-11eb-87ad-06833ad1c665.png)

Service logs
All logs
January 10, 2021
5:27 PM
Failed to start Mount ZFS filesystems.
systemd
5:27 PM
zfs-mount.service: Failed with result 'signal'.
systemd
5:27 PM
zfs-mount.service: Main process exited, code=killed, status=6/ABRT
systemd
5:27 PM
free(): double free detected in tcache 2
zfs
5:27 PM
cannot mount '/tank': directory is not empty
zfs
5:27 PM
cannot mount '/rpool': directory is not empty
zfs
5:27 PM
cannot mount '/logdisc': directory is not empty
zfs
5:27 PM
Starting Mount ZFS filesystems...

```

Failed to start Mount ZFS filesystems.
--
CODE_FILE | ../src/core/job.c
CODE_FUNC | job_log_done_status_message
CODE_LINE | 950
INVOCATION_ID | c7c8fee1e7e74aeca8eb01bb679684a4
JOB_ID | 1856882
JOB_RESULT | failed
JOB_TYPE | start
MESSAGE_ID | be02cf6855d2428ba40df7e9d022f03d
PRIORITY | 3
SYSLOG_FACILITY | 3
SYSLOG_IDENTIFIER | systemd
UNIT | zfs-mount.service
_BOOT_ID | 12194bde86f94080ba5e821c87e4c0cf
_CAP_EFFECTIVE | 3fffffffff
_CMDLINE | /sbin/init
_COMM | systemd
_EXE | /lib/systemd/systemd
_GID | 0
_HOSTNAME | pve
_MACHINE_ID | 5a2bd0d11a6c41f9a33fd527751224ea
_PID | 1
_SELINUX_CONTEXT | unconfined
_SOURCE_REALTIME_TIMESTAMP | 1610296033434024
_SYSTEMD_CGROUP | /init.scope
_SYSTEMD_SLICE | -.slice
_SYSTEMD_UNIT | init.scope
_TRANSPORT | journal
_UID | 0
__CURSOR | s=6a480fdb55a04797bf197f3735b72afd;i=5bea6;b=12194bde86f94080ba5e821c87e4c0cf;m=17a54f7e86d;t=5b88e430e59c9;x=f21100dea5dcfe73
__MONOTONIC_TIMESTAMP | 1624923170925
__REALTIME_TIMESTAMP | 1610296033434057


```

**Comment by Ondjultomte** [comment]
![image](https://user-images.githubusercontent.com/16661547/104128988-d3347000-536a-11eb-8d20-7e48ac5c77d9.png)

**Comment by iamoptimans** [comment]
Hi @Ondjultomte,

Does the Cockpit ZFS Manager package work within Cockpit?

These looks like separate issues. The first issue looks like you need to enable the overlay parameter when auto mounting. They fixed this in OpenZFS 2.0.

Not sure about udisks2, as it is not related to CZM.
